contrib/ec2/gen-json.py

```diff
@@ -60,13 +65,20 @@
   RemainAfterExit=yes
   ExecStart=/usr/bin/chown -R etcd:etcd /media/etcd
 '''
+ETCD_DROPIN = '''
```

Comment:
is etcd socket-activated on EC2 as well? Just curious why this is necessary since the PR didn't say anything about etcd